libafl_nyx/src/settings.rs

```diff
@@ -0,0 +1,23 @@
+use typed_builder::TypedBuilder;
+
+const DEFAULT_INPUT_BUFFER_SIZE: u32 = 1024 * 1024;
+const DEFAULT_TIMEOUT_SECS: u8 = 2;
+const DEFAULT_TIMEOUT_MICRO_SECS: u32 = 0;
+
+#[derive(Debug, Clone, Copy, TypedBuilder)]
+pub struct NyxSettings {
```

Comment:
If anyone should feel like it, it could be nice to have some doc comments for these fields